tripy/tripy/utils/utils.py

```diff
@@ -468,3 +468,29 @@ def merge_function_arguments(func, *args, **kwargs):
     all_args = get_positional_arg_names(func, *args)
     all_args.extend(kwargs.items())
     return all_args
+
+
+def get_arg_by_name(name, func, *args, **kwargs):
```

Comment:
Can you add some docstrings for both `get_arg_by_name` and `modify_arg` (makes it easier to reuse these utils).

Reply:
Added to `modify_arg`. I think `get_arg_by_name` is fairly self-explanatory.